pkgs/tools/package-management/nix/default.nix

```
@@ -182,6 +182,20 @@ in lib.makeExtensible (self: {
     sha256 = "sha256-hNHfvmb1bIWwqFT5nesQgwh4V0OlyZHxj5ZVSQbZ+p4=";
   };
 
+  # The minimum Nix version supported by Nixpkgs
```

```suggestion
  # The minimum Nix version supported by Nixpkgs
  # Note that some functionality *might* have been backported into this Nix version,
  # making this package an inaccurate representation of what features are available
  # in the actual lowest minver.nix *patch* version.
```

We generally don't backport a lot, and probably 2.3.0(1?) is not relevant anymore anyway, but testing against this may give a false sense of ~~security~~ correctness.